webui/react/src/e2e/tests/userManagement.spec.ts

```diff
@@ -36,41 +38,33 @@ test.describe('User Management', () => {
     }
   });
 
-  test('Navigate to User Management', async ({ page }) => {
-    const userManagementPage = new UserManagement(page);
+  test('Navigate to User Management', async ({ authedPage }) => {
+    const userManagementPage = new UserManagement(authedPage);
     await (await userManagementPage.nav.sidebar.headerDropdown.open()).admin.pwLocator.click();
-    await expect(page).toHaveTitle(userManagementPage.title);
-    await expect(page).toHaveURL(userManagementPage.url);
+    await expect(authedPage).toHaveTitle(userManagementPage.title);
+    await expect(authedPage).toHaveURL(userManagementPage.url);
   });
 
   test.describe('With New User Teardown', () => {
-    test.afterAll(async ({ browser }) => {
-      const pageSetupTeardown = await browser.newPage();
-      const authFixtureSetupTeardown = new AuthFixture(pageSetupTeardown);
-      const userFixtureSetupTeardown = new UserFixture(pageSetupTeardown);
-      const userManagementPageSetupTeardown = new UserManagement(pageSetupTeardown);
-      await authFixtureSetupTeardown.login();
+    test.afterAll(async ({ backgroundApiUser }) => {
+      await backgroundApiUser.apiAuth.login();
       await test.step('Deactivate Users', async () => {
-        await userManagementPageSetupTeardown.goto();
-        await userFixtureSetupTeardown.deactivateAllTestUsers();
+        for (const [id] of testUsers) {
+          await backgroundApiUser.apiAuth.login();
```

Comment:
q: why do you need to login with the `backgroundApiUser` at the beginning of this section and then also for each test user?

Reply:
I don't think it should be, good catch!